### PR TITLE
Social icons

### PR DESCRIPTION
**Pull request by JakeSidSmith**
screen shot 2018-03-08 at 12 35 49
screen shot 2018-03-08 at 12 35 57
screen shot 2018-03-08 at 12 36 03

### CODE REVIEW

**Comment by yanneves** [comment]
changes look solid, however, I wonder whether we're starting to lose a lot of vertical space to the masthead - not sure what our ratio is on the site between new and existing users - worth some thought

File: _includes/partials/masthead.html [hunk 1]
```diff
@@ -2,4 +2,13 @@
   <img class="logo" src="/img/logo-100x100.png" width="50" height="50" />
   <h1><a href="/">{{ site.title }}:</a></h1>
   {{{ markdown site.caption }}}
+  <ul class="social">
+    {{#each site.social as |social|}}
+      <li>
+        <a href="{{ social.link }}" title="{{ social.desc }}">
```
[the remark this answers is not in the document]
Comment (JakeSidSmith) [reply _includes/partials/masthead.html]: Ah, yeah, good point. 🙂

**Comment by JakeSidSmith** [comment]
I think it is fine as it is now, but could do with making a bit smaller. As long as you can see the top of the content on mobile I think it's fine. :)

**Comment by JakeSidSmith** [comment]
I intend to make some big changes to the masthead at some point, but for now I thought it'd be good to move these to the top as I believe a lot of people miss them.

File: _assets/styles/global.scss [hunk 2]
```diff
@@ -55,13 +58,14 @@ abbr {
 a {
   color: $black;
   text-decoration: none;
-  -webkit-transition: background-color, border-color, color;
+  -webkit-transition: background-color, border-color, color, fill;
```
Comment (yanneves) [inline _assets/styles/global.scss]: no need for the browser prefixes, postcss/autoprefixer has got your back! 😎

**Comment by yanneves** [comment]
just a minor comment above, looks good to merge otherwise, nice!

File: _assets/styles/global.scss (hunk 2 above)
Comment (JakeSidSmith) [reply _assets/styles/global.scss]: These were already here. So I left 'em.

Comment (yanneves) [reply _assets/styles/global.scss]: ah, I see

Comment (yanneves) [reply _assets/styles/global.scss]: right, indeed, got loads of browser prefixes hanging around - let me pick this up under a new PR

**Pull request merged**